packages/panna-sdk/src/react/components/buy/select-buy-provider-step.tsx

```diff
@@ -65,47 +59,80 @@ export function SelectBuyProviderStep({ form }: SelectBuyProviderStepProps) {
     fiatCurrency: currencyCode
   });
 
-  const handleProviderSelect = async (
+  const providerFromQuote = useMemo(() => {
+    if (!quote) {
+      return null;
+    }
+
+    const providerId = quote.provider_id ?? 'onramp-money';
+    const matchedProvider = availableProviders.find(
+      (provider) => provider.id === providerId
+    );
+
+    return matchedProvider ?? null;
```

Comment:
Are we now returning only one quote? We should return a list of all the available provider quotes instead.